apps/src/codebridge/redux/consoleRedux.ts

```diff
@@ -5,7 +5,13 @@ export interface CodeBridgeConsoleState {
 }
 
 export interface ConsoleLog {
-  type: 'system_out' | 'system_in' | 'img' | 'system_msg' | 'error';
+  type:
+    | 'system_out'
+    | 'system_in'
+    | 'img'
+    | 'system_msg'
+    | 'error'
+    | 'system_error';
```

Comment:
totally not worth waiting for test: but could be nice to alphabetically sort these 🤷

Reply:
I'll try and sneak that into another pr